enhancements/kube-apiserver/audit-policy.md

```diff
@@ -152,6 +152,7 @@ we provide the following profiles:
 
 - `WriteRequestBodies`: this is like `Default`, but it logs request and response HTTP payloads for write requests (create, update, patch).
 - `AllRequestBodies`: this is like `WriteRequestBodies`, but also logs request and response HTTP payloads for read requests (get, list).
+- `UserRequests`: this logs the requests from users in the `system:authenticated:oauth` user groups and logs requests for (create, delete).
```

Comment:
we might want to be more specific: `OAuthAuthenticated`. Opinions?

Reply:
Also `UserRequests` is not in-line with the other three. Do we mean `UserRequestBodies`? Also it does not really say whether other requests stay in metadata level. This is the discussion we started last week.

Reply:
Name is now updated to 'UserRequestBodies` in both the Audit Policy in `library-go` and API in `openshift-api` for the Enum to set also